docs/HowToGuides/FAQ.md

```diff
@@ -133,5 +133,4 @@ git rebase --continue
 
 ### How do I clean up my git history?
 
-TODO: Link to a beginner-friendly external resource, or (less preferably)
-describe basic usage of rebase here.
+[Link to the guide](https://medium.com/@catalinaturlea/clean-git-history-a-step-by-step-guide-eefc0ad8696d)
```

Comment:
```suggestion
See [GitLab's guide to keeping your git commit history clean](https://about.gitlab.com/blog/2018/06/07/keeping-git-commit-history-clean/).
```

I think this resource has more details, with a helpful picture as well explaining how rebase works.